examples/jupyter/demo010_advection.ipynb

```diff
@@ -149,18 +97,10 @@
     "    # compute\n",
     "    \n",
     "\n",
-<<<<<<< HEAD
     "    (_, eigenvalues, _) = computesymbolsoverrange(advection, numbersteps, mass=mass)\n",
     "    maxeigenvalues = reshape(maximum(real(eigenvalues); dims=2), (numbersteps, numbersteps))\n",
     "    mineigenvalues = reshape(minimum(real(eigenvalues); dims=2), (numbersteps, numbersteps))\n",
     "\n",
-=======
-    "    (_, eigenvalues, _) = computesymbolsoverrange(advection, numbersteps, mass=mass, θ_min = 0., θ_band = 1π)\n",
-    "    # Flip the sign because the weak form is the negative advection operator\n",
-    "    maxeigenvalues = reshape(maximum((-imag(eigenvalues)); dims=2), (numbersteps, numbersteps))\n",
-    "    mineigenvalues = reshape(minimum((-imag(eigenvalues)); dims=2), (numbersteps, numbersteps))\n",
-    "    \n",
->>>>>>> 48dc89b931d819c8a441fe77fbbf998b0a57a77b
```

Comment:
Here you've removed the code that's necessary to plot the advection operator (which has imaginary eigenvalues).